src/celeritas/ext/detail/ScopedRootErrorHandler.hh

```diff
@@ -0,0 +1,45 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2022 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file celeritas/ext/detail/RootErrorHandler.hh
```

Comment:
`detail` classes should be use only (perhaps with a very few exceptions) by the parent directory. Since this is going to be used in the demo loops etc., it should be in `ext/` and not in a `detail` namespace.

Reply:
ok.